bucket/wizfile.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "2.06",
+    "description": "A file search utility that search for files on your hard drive by name.",
+    "homepage": "https://antibody-software.com/web/software/software/wizfile-finds-your-files-fast/",
+    "license": "Unknown",
+    "url": "https://antibody-software.com/files/wizfile_2_06_portable.zip",
+    "hash": "5c340e11fef0b23b213d0aaa41a18695b5b96235c24bde0fd4d32582d2c9a8d4",
+    "pre_install": "if (-not (Test-Path \"$persist_dir\\WizFile.ini\")) { New-Item \"$dir\\WizFile.ini\" -ItemType File | Out-Null }",
+    "architecture": {
+        "64bit": {
+            "bin": [
```

```suggestion
            "bin": [
                “WizFile64.exe”,
                [
                    "WizFile.exe",
                    "WizFile32"
                ],
```
In general, I suggest we give users the option to explicitly run the 64-bit or 32-bit versions, ok?